### PR TITLE
Test hygiene: de-flake guard-time-trip + untrack generated debugger fixtures

### LINKED ISSUE #467: Flaky test: guard-time-trip 2ms tolerance fails on loaded CI runners
## Symptom

`tests/agency/guards/guard-time-trip` failed on CI (`agency-tests (23.x)`, run 29019325761, on PR #466):

```
- "tripped:timeoutFailure:20:true"
+ "tripped:timeoutFailure:20:false"
```

The trailing boolean is the assertion that `actualTime` lands within **2ms** of the configured 20ms budget. On a loaded shared runner the abortable sleep woke late, so the tolerance check failed. The identical commit passed on the 22.x leg, passes 3/3 locally, and the compiled output on that PR is proven byte-identical to main — this is scheduler jitter, not a code regression.

## History

This test has already been de-flaked once: 9fe490ad ("Fix flaky guard-time-trip test: tolerate clock-source skew") introduced the current 2ms tolerance for performance.now()-vs-timer skew. The new failure shows the remaining flake source is runner scheduling latency, which no small fixed tolerance can absorb.

## Suggested fix

The property the test actually cares about is "the time guard trips, reports `type: timeoutFailure`, and carries the configured `maxTime`" — not "the runner scheduler is punctual". Either:

1. Assert `actualTime >= maxTime` only (the guard can never legitimately trip early), dropping the upper bound entirely; or
2. Widen the tolerance to something CI-realistic (e.g. 25–50ms) if an upper bound is worth keeping as a smoke check against a hung-then-recovered timer.

Option 1 is the tighter statement of the invariant and immune to scheduling jitter.

## References

- Failing run: https://github.com/egonSchiele/agency-lang/actions/runs/29019325761 (agency-tests 23.x, Agency execution tests step)
- Test: `packages/agency-lang/tests/agency/guards/guard-time-trip.agency` / `.test.json`
- Prior de-flake: 9fe490ad

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### PR DESCRIPTION
**Pull request by egonSchiele**
Two small test-hygiene fixes, both previously flagged:

## De-flake guard-time-trip (fixes #467)

The test asserted `actualTime >= maxTime - 2`. The 2ms allowance was the previous de-flake for clock-source skew (`performance.now()` vs the libuv timer), but loaded CI runners add scheduling latency on top and the test flaked at exactly that boundary (PR #466 CI run). The property the assertion protects — per its own comment — is that `actualTime` is a real measurement rather than a 0/sentinel value. It now requires `actualTime >= maxTime / 2`: immune to any plausible skew, still catches a broken measurement. Expected output unchanged; passes 3/3 locally.

## Untrack the generated debugger fixtures (flagged on PR #466)

`tests/debugger/*.ts` are build artifacts: every consumer (`driver.test.ts`, `exportFrames.test.ts`, `testSession.test.ts`) regenerates them in `beforeAll`, and `driver.test.ts`'s comment says they are gitignored. They were tracked anyway because the ignore pattern contained a slash and therefore anchored to the repo ROOT — it never matched `packages/agency-lang/tests/debugger/`. Root cause fixed (`**/` prefix, with a comment explaining the anchoring trap), six files untracked. Full local vitest runs no longer dirty the tree whenever codegen output drifts from the committed snapshot.

Verified: debugger + tui suites pass from a clean slate (fixtures deleted first, regenerated by beforeAll); regenerated files are properly ignored; lint clean.

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: packages/agency-lang/tests/agency/guards/guard-time-trip.agency [hunk 1]
```diff
@@ -16,13 +16,14 @@ node main() {
...
-    // on sub-millisecond clock skew.
-    return "tripped:${result.error.type}:${result.error.maxTime}:${result.error.actualTime >= result.error.maxTime - 2}"
+    // timer fires off libuv's coarser millisecond clock, and loaded CI
+    // runners add scheduling latency on top — the two readings can
+    // disagree by MORE than the 2ms this test used to allow (issue
+    // #467: it flaked at exactly that boundary). The property worth
+    // asserting is that actualTime is a real measurement rather than a
+    // 0/sentinel value, so require it to reach half the budget: immune
+    // to any plausible clock skew, still catches a broken measurement.
+    return "tripped:${result.error.type}:${result.error.maxTime}:${result.error.actualTime >= result.error.maxTime / 2}"
```
[the remark this answers is not in the document]
Comment (egonSchiele) [reply packages/agency-lang/tests/agency/guards/guard-time-trip.agency:27]: Fixed in f0f8a8c7 — the module comment now describes the loose half-budget assertion and points at the inline rationale.
